ballerina/types.bal

```diff
@@ -14,172 +14,550 @@
+# + token - Unique identifier created by the notification service for an app on a device. The specific name for 
+#           the token will vary, depending on which notification service is being used
+public type EndpointAttributes record {|
+    string customUserData?;
+    boolean enabled?;
+    string token?;
+|};
+
+# Represents an Amazon SNS platform appication endpoint.
+# 
+# + endpointArn - The endpoint's ARN
+public type Endpoint record {|
+    string endpointArn;
+    *EndpointAttributes;
+|};
+
+# Represent an SMS sandbox phone number.
+# 
+# + phoneNumber - The destination phone number
+# + status - The destination phone number's verification status
+public type SMSSandboxPhoneNumber record {|
+    string phoneNumber;
+    Status status;
+|};
+
+# Represents an origination phone number.
+# 
+# + createdAt - The date and time when the origination phone number was created
+# + iso2CountryCode - The two-character code, in ISO 3166-1 alpha-2 format, for the country or region where the
+#                     origination phone number was originally registered
+# + numberCapabilities - The capabilities of the origination phone number
+# + phoneNumber - The phone number
+# + routeType - The route type
+# + status - The status of the origination phone number
+public type OriginationPhoneNumber record {|
+    time:Civil createdAt;
+    string iso2CountryCode;
+    NumberCapabilities[] numberCapabilities;
+    string phoneNumber;
+    RouteType routeType;
+    string status;
+|};
+
+# Represents the attributes for sending SMS messages with Amazon SNS.
+# 
+# + monthlySpendLimit - The maximum amount in USD that you are willing to spend each month to send SMS messages. When
+#                       Amazon SNS determines that sending an SMS message would incur a cost that exceeds this limit,
+#                       it stops sending SMS messages within minutes
+# + deliveryStatusIAMRole - The ARN of the IAM role that allows Amazon SNS to write logs about SMS deliveries in
+#                           CloudWatch logs
+# + deliveryStatusSuccessSamplingRate - The percentage of successful SMS deliveries for which Amazon SNS will write
+#                                       logs in CloudWatch Logs
+# + defaultSenderID - A string that is displayed as the sender on the receiving device
+# + defaultSMSType - The type of SMS message that you will send by default
+# + usageReportS3Bucket - The name of the Amazon S3 bucket to receive daily SMS usage reports from Amazon SNS
+public type SMSAttributes record {|
+    int monthlySpendLimit?;
+    string deliveryStatusIAMRole?;
+    int deliveryStatusSuccessSamplingRate?;
+    string defaultSenderID?;
+    SMSMessageType defaultSMSType?;
+    string usageReportS3Bucket?;
+|};
+
+# Represents the tags associated with an Amazon SNS topic.
+# 
+# + topicArn - The ARN of the topic to which the tags are added
+public type Tags record {|
+    never topicArn?;
+    string...;
+|};
```

Comment:
```suggestion
|};

```

Reply:
Fixed in [020b0a4](https://github.com/ballerina-platform/module-ballerinax-aws.sns/pull/40/commits/020b0a40d31509c3e7b499cf2ea5c620bbfc9b2b)